src/components/mgt-person/mgt-person.ts

```diff
@@ -527,4 +555,12 @@ export class MgtPerson extends MgtTemplatedComponent {
 
     return initials;
   }
+
+  private numberFromInitials(initials) {
```

Comment:
Using only initials will generate the same color for the same initials even if the names are different. It's important that we try to have a different color, especially if the names are the same, to make it obvious when there are different people. Because of that, I propose to generate the index by prioritizing id and then displayName. If id is available, use it for the hash, otherwise, use the full name. Don't use initials

Reply:
Also, I'd change this method to actually return the classes to make it more readable

Reply:
done